recipes/jupyter-forward/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "jupyter-forward" %}
+{% set version = "2020.10.27" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8a0491f5a41c28cd81ee6febdf645162925caf88c83a58bbc9240281c23fef9d
+
+build: 
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
```

Comment:
If you use grayskull to generate the recipe it will get the updated script line for you.

https://pypi.org/project/grayskull/

Reply:
Thank you for the pointer to grayskull, @ocefpaf! I'll be using it in the future.